app/src/androidTest/java/ch/hikemate/app/ui/profile/ProfileScreenTest.kt

```diff
@@ -32,7 +32,7 @@ import org.mockito.kotlin.times
 import org.mockito.kotlin.verify
 
 class ProfileScreenTest : TestCase() {
-  private lateinit var authRepository: AuthRepository
+  private lateinit var authRepository: FirebaseAuthRepository
```

Comment:
Warning: I think you missed merge here. You should keep the `authRepository: AuthRepository`